x-pack/plugins/security_solution/server/lib/entity_analytics/risk_engine/risk_engine_data_client.ts

```diff
@@ -88,13 +68,23 @@ export class RiskEngineDataClient {
     }
 
     try {
-      await this.initializeResources({ namespace });
+      await riskScoreDataClient.init();
       result.riskEngineResourcesInstalled = true;
     } catch (e) {
       result.errors.push(e.message);
       return result;
     }
 
+    if (isAssetCriticalityEnabled) {
+      try {
+        await assetCriticalityDataClient.init();
```

Comment:
We are now only creating the asset criticality index when someone enables the risk engine, right?

Does this mean that existing users who have already enabled the risk engine must disable then re-enable the engine in order to have the asset criticality index created?

Reply:
In this PR yes only when you enable risk engine.
We also have a task for upgrading workflow, exactly for existing clients